mne_bids/tests/test_utils.py

```diff
@@ -71,3 +76,89 @@ def test_age_on_date():
     assert age_on_date(bday, exp3) == 24
     with pytest.raises(ValueError):
         age_on_date(bday, exp4)
+
+
+def test_get_brainvision_paths():
```

Comment:
you didn't double underscore ;)

Reply:
are we doing that? @jasmainak wasn't too impressed when I tried to do it the last time :-D

Reply:
I tried to find some conventions but the only one i could find was to name the tests [`test_***`](https://pytest.readthedocs.io/en/reorganize-docs/new-docs/user/naming_conventions.html). So I changed the `test__***` to simply be `test_***` ... because that double underscore does look a bit silly

Reply:
In mne, we use the single underscore :)